applications/nrf_desktop/src/util/dfu_flash_lock.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef _DFU_FLASH_LOCK_H_
+#define _DFU_FLASH_LOCK_H_
+
+#include <limits.h>
+#include <assert.h>
+#include <sys/atomic.h>
+
+/* This function should be called in system workqueue context. It's recommended
+ * to call it during module initialization.
+ */
+atomic_val_t dfu_flash_get_key(void);
```

Comment:
I think it is better to get rid of this function and use module ids instead.

Reply:
ok